src/Sentry/SentryClient.cs

```diff
@@ -82,6 +82,42 @@ public SentryId CaptureEvent(SentryEvent? @event, Scope? scope = null, SentryHin
     }
 
     /// <inheritdoc />
+    public void CaptureFeedback(SentryFeedback feedback, Scope? scope = null, SentryHint? hint = null)
+    {
+        if (string.IsNullOrEmpty(feedback.Message))
+        {
+            _options.LogWarning("Feedback dropped due to empty message.");
+            return;
+        }
+
+        scope ??= new Scope(_options);
+        hint ??= new SentryHint();
+        hint.AddAttachmentsFromScope(scope);
+
+        _options.LogInfo("Capturing event.");
+
+        var evt = new SentryEvent { Level = SentryLevel.Info };
+        evt.Contexts.Feedback = feedback;
```

Comment:
I find this odd. Are we trying to find a way to attach the feedback to the event somehow? 
Could the feedback inherit from SentryEvent instead?

Reply:
Not without changing all the other SDKs. This is just how the spec says we should be doing it.